cmd/couchbase-database-plugin/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	hclog "github.com/hashicorp/go-hclog"
```

Comment:
The ordering of these import lines isn't right, normally standard library includes are on their own first. Same goes for a couple of other files too (`httputils.go` and `couchbase_test.go`). Most people set up some run-on-save hooks for things like [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports?tab=doc) so that this stuff is just done automatically.

If you'd like to just run it manually this time, `goimports -w .` from the root directory should sort everything out.

Reply:
Done.